### PR TITLE
Add travis configuration.

### PR DESCRIPTION
**Pull request by korneliakobiela**

### CODE REVIEW

**Comment by singa2000** [comment]
@hsiwkin ,@lslachciak 
Would you review, please?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,19 @@
+os: 
+  - linux
+  - windows
+
```
Comment (singa2000) [inline .travis.yml]: How about the Mac?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+language: node_js
+node_js: 
+  - "10"
+
+jobs:
+  include:
+    - stage: "WATT"
```
Comment (hsiwkin) [inline .travis.yml]: Maybe WATT-WINDOWS would be more consistent with the other stage names? 
Apart from that let's be consistent in keys order (eg stage, script, os).

File: .travis.yml (hunk 1 above)
Comment (korneliakobiela) [reply .travis.yml]: Are we support our extensions for Mac?

File: .travis.yml (hunk 2 above)
Comment (korneliakobiela) [reply .travis.yml]: AFAIR WATT works only in Linux server environment. @lslachciak Am I right? 
I agree with the second part of your comment.

Comment (lslachciak) [reply .travis.yml]: Currently Open Source WATT CI check is only enabled for Ubuntu. There were some problems with CI run on Mac, so I assume also that WATT is not supported on Mac now.

File: .travis.yml (hunk 1 above)
Comment (singa2000) [reply .travis.yml]: Design Editor need to support the Mac. WAPPL and TAUrine also support the MaC.
But I'm not sure travis also support the Mac os.
If the travis don't support the Mac os, you can skip it.

**Comment by korneliakobiela** [comment]
Closes #17

**Pull request merged**